integration_tests/commands/async/deque_test.go

```diff
@@ -121,17 +121,17 @@ func TestRPush(t *testing.T) {
 	}{
 		{
 			name:   "RPUSH",
-			cmds:   []string{"LPUSH k v", "LPUSH k v1 1 v2 2", "LPUSH k 3 3 3 v3 v3 v3"},
+			cmds:   []string{"RPUSH k v", "RPUSH k v1 1 v2 2", "RPUSH k 3 3 3 v3 v3 v3"},
 			expect: []any{int64(1), int64(5), int64(11)},
 		},
 		{
 			name:   "RPUSH normal values",
-			cmds:   []string{"LPUSH k " + strings.Join(deqNormalValues, " ")},
+			cmds:   []string{"RPUSH k " + strings.Join(deqNormalValues, " ")},
 			expect: []any{int64(25)},
 		},
 		{
 			name:   "RPUSH edge values",
-			cmds:   []string{"LPUSH k " + strings.Join(deqEdgeValues, " ")},
+			cmds:   []string{"RPUSH k " + strings.Join(deqEdgeValues, " ")},
```

Comment:
Why have we replaced `LPUSH` with `RPUSH`? Ideally these should be added as new testcases instead of replacing the older ones.

Reply:
There was a mistake in this function. The function above this is testing LPUSH and this is supposed to be testing RPUSH,
but the RPUSH test also had LPUSH commands only,
so i changed it.